docker-compose.yaml

```diff
@@ -34,6 +30,5 @@ services:
       REGISTRY_AUTH_TOKEN_ISSUER: Registry token issuer
       REGISTRY_AUTH_TOKEN_ROOTCERTBUNDLE: /root/certs/bundle
     volumes:
-      - /telescope/data:/var/lib/registry
-      - /telescope/certs:/certs
-      - /telescope/auth:/auth
+      - /mnt/docker0storage/registry:/var/lib/registry
```

Comment:
We'll have to remember to create this directory and give it the right permissions.